src/bin/bombadil.rs

```diff
@@ -17,7 +17,7 @@ macro_rules! fatal {
     }}
 }
 
-/// Toml is a dotfile template manager, written in rust.
+/// Toml Bombadil is a dotfile template manager, written in rust.
```

Comment:
I don't actually know if this ends up in the `--help` output, but it was the only git GH search gave me about this string :)

```shell
$ bombadil --help | head -n 1
Toml is a dotfile template manager, written in rust
```